packages/.vscode/launch.json

```diff
@@ -0,0 +1,59 @@
+{
+    // Adjusted copy of devtools_app/.vscode/launch.json,
```

Comment:
@kenzieschmoll what do you think of making this the default config? Seems like everyone should be seeing errors in the devtools_test package.

Reply:
In this PR `settings.json` updates working directory to devtools_app, to make console experience the same as in devtools_app.